lib/features/mailbox_dashboard/presentation/widgets/navigation_bar/navigation_bar_widget.dart

```diff
@@ -66,15 +58,6 @@ class NavigationBarWidget extends StatelessWidget {
                   child: searchForm
                 ),
                 const Spacer(),
-                if (contactSupportCapability?.isAvailable == true)
-                  TMailButtonWidget.fromIcon(
```

Comment:
please still keep contact support button in app bar

Reply:
Updated


https://github.com/user-attachments/assets/c5b60a21-fcc2-4207-b536-6c0885e8203f